docs/cloud-native-security/kspm.asciidoc

```diff
@@ -1,2 +1,38 @@
 [[kspm]]
 = Kubernetes security posture management
+
+The Kubernetes Security Posture Management (KSPM) integration allows you to monitor how your Kubernetes clusters' configuration measures up to security benchmarks.
+
+To set up the integration, you'll need to first add it to an agent policy, then deploy the KSPM DaemonSet to the Kubernetes clusters you want to monitor.
+
+[discrete]
+== Set up a KSPM integration
+To install the integration:
+
+1. Go to *Dashboards -> Cloud Posture*
+2. Click *Add a CIS integration*.
+3. Click *Add Kubernetes Security Posture Management*.
+4. Name your integration.
+5. Select whether to use the CIS or EKS Benchmarks — use CIS unless you're deploying on EKS.
```

Comment:
I'm leaning no, because most people will either know, or all they really need to know is what the second half of the sentence says: use CIS unless you're on EKS.